homework-submission/Week1/App.js

```diff
@@ -0,0 +1,72 @@
+const ListItem = ({ description, deadline, className }) => {
+  return (
+    <li class={className}>
+      {description}, {deadline}
+    </li>
+  );
+};
+// The goal of this component is to create a single item of a list .
+
+const StaticList = () => {
+  return (
+    <div>
+      <ul>
+        {<ListItem description="Get out of bed" deadline="Wed Sep 13 2017" />}
+        {<ListItem description="Brush teeth" deadline="Thu Sep 14 2017" />}
+        {<ListItem description="Eat breakfast" deadline="Fri Sep 15 2017" />}
+      </ul>
+    </div>
+  );
+};
+// This StaticList function creates a ul list component which has its own props.
```

Comment:
Kindly comments, regarding to there spot what about writing them above the function or in the first line, so they will be meant for it for somebody reads the code for his first time.

That would be helpful especially when you have a long function.